documentation/docs/libraries/wasm/cheat_sheet.md

```diff
@@ -1,547 +0,0 @@
----
-title:  Wasm Cheat Sheet
-sidebar_label: Cheat Sheet
```

Comment:
Should we remove the Rust cheat sheet too?

Reply:
Yeah, it's also outdated.